### PR TITLE
add `download_instructions` to ORCA easyconfigs 

### PR DESCRIPTION
**Pull request by branfosj**
for #19881

### CODE REVIEW

File: test/easyconfigs/easyconfigs.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def get_files_from_diff(diff_filter, ext):
     # first determine the 'merge base' between target branch and PR branch
     # cfr. https://git-scm.com/docs/git-merge-base
     cmd = "git merge-base %s HEAD" % target_branch
-    res = run_shell_cmd(cmd, fail_on_error=False)
+    res = run_shell_cmd(cmd, fail_on_error=False, hidden=True)
```
Comment (lexming) [inline test/easyconfigs/easyconfigs.py:114]: these changes to the tests seem unintended for this PR, are they needed?

**Review by lexming — CHANGES_REQUESTED** [review]

File: test/easyconfigs/easyconfigs.py (hunk 1 above)
Comment (Micket) [reply test/easyconfigs/easyconfigs.py:114]: Looks like this slipped in from this PR?
https://github.com/easybuilders/easybuild-easyconfigs/pull/19886

Comment (branfosj) [reply test/easyconfigs/easyconfigs.py]: fixed

**Review by Micket — APPROVED** [review]
lgtm

**Pull request merged**